cmake/BLASConfig.cmake

```diff
@@ -3,16 +3,20 @@
 # This program is free software: you can redistribute it and/or modify it under
 # the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
 
-# Check if this file has already been run with these settings.
+include( "cmake/util.cmake" )
```

Comment:
I am somewhat confused that this was not included prior. But the def's defined there are used in the routine already. Does CMake automatically find the file or would those just never be used correctly here?

Reply:
It was included in line 15 before. It moved.